models/public/mobilefacedet-v1-mxnet/mobilefacedet-v1-mxnet.md

```diff
@@ -0,0 +1,100 @@
+# mobilefacedet-v1-mxnet
+
+## Use Case and High-Level Description
+
+  MobileFace Detection V1 is a Light and Fast Face Detector for Edge Devices (LFFD) model based on Yolo V3 architecture and trained with MXNet. For details see the [repository](https://github.com/becauseofAI/MobileFace) and [paper](https://arxiv.org/pdf/1904.10633.pdf).
+
+## Specification
+
+| Metric            | Value         |
+|-------------------|---------------|
+| Type              | Detection     |
+| GFLOPs            | 3.5456       |
+| MParams           | 7.6828        |
+| Source framework  | MXNet\*       |
+
+## Input
+
+### Original model
+
+Image, name - `data`, shape - `[1x256x256x3]`, format -`[BxHxWxC]` where:
```

Comment:
I see, the BHWC->BCHW conversion only applies to TF model, am i right?

Reply:
Model contains preprocessing node inside which transpose NHWC->NCHW  and it is not possible to remove (I asked MO team about it, they said no way)

Reply:
The original model is not converted to BCHW format. This model takes input in a  BHWC format and converts the input data within itself.

Reply:
Ok, fine. But I strongly recommend you to add description of converted model input with some 
>**WARNING**

sign, because most of models n OMZ has NCHW layout

Reply:
please review again